docker/pygeoapi-config.yml

```diff
@@ -26,70 +26,31 @@ logging:
 metadata:
     identification:
         title:
-            en: WIS 2.0 in a box
+            en: wis2box-api
         description:
             en: |-
-                WIS 2.0 in a box provides a platform with the capabilities
+                WIS 2.0 in a box (wis2box) provides a platform with the capabilities
                 for centres to integrate their data holdings and publish them
                 to the WMO Information System with a plug and play capability
-                supporting data publishing, discovery and access
+                supporting data publishing, discovery and access.
+                The wis2box-api provides processes to manage datasets and tools
+                for data transformation and publication.
         keywords:
             en:
-                - wmo
-                - wis 2.0
-        keywords_type: theme
-        terms_of_service: https://public.wmo.int/en/our-mandate/what-we-do/observations/Unified-WMO-Data-Policy-Resolution
-        url: https://community.wmo.int/activity-areas/wis/wis2-implementation
-    license:
-        name: Unified WMO Data Policy
-        url: https://public.wmo.int/en/our-mandate/what-we-do/observations/Unified-WMO-Data-Policy-Resolution
+                - wis2box
+        terms_of_service: none
+        url: none
     provider:
-        name: Organization Name
-        url: https://pygeoapi.io
+        name: WIS 2.0 in a box
+        url: https://docs.wis2box.wis.wmo.int
     contact:
-        name: Lastname, Firstname
-        position: Position Title
-        address: Mailing Address
-        city: City
-        stateorprovince: Administrative Area
-        postalcode: Zip or Postal Code
-        country: Country
-        phone: +xx-xxx-xxx-xxxx
-        fax: +xx-xxx-xxx-xxxx
-        email: you@example.org
-        url: https://example.org
-        hours: Mo-Fr 08:00-17:00
-        instructions: During hours of service. Off on weekends.
-        role: pointOfContact
+        name: none
+        email: none
+    license:
```

Comment:
This should point to information for licensing; the WMO Unified Data Policy would be the safest value to encompass datasets published through wis2box (instance wide).  Note that this value is displayed on the API landing page.

Reply:
@tomkralidis can you please provide a proposal that would be a valid placeholder for anyone running a wis2box ?

Reply:
Below, for consideration/feedback:

```yaml
    identification:
        title:
            en: WIS2 in a box API
        description:
            en: |-
                WIS 2.0 in a box (wis2box) provides a platform with the capabilities
                for centres to integrate their data holdings and publish them
                to the WMO Information System with a plug and play capability
                supporting data publishing, discovery and access.
                wis2box-api provides an API providing data and processes for interactive
                visualization, data transformation and publication.
        keywords:
            en:
                - wis2box
                - WIS2 Node
        url: https://docs.wis2box.wis.wmo.int
    provider:
        name: WIS 2.0 in a box
        url: https://docs.wis2box.wis.wmo.int
    contact:
        name: World Meteorological Organization
        address: 7bis, avenue de la Paix
        city: Geneva
        postalcode: 1211
        country: Switzerland
        email: null
        url: https://wmo.int/contact-us
        role: pointOfContact
    license:
        name: WMO Unified Data Policy
        url: https://library.wmo.int/records/item/58009-wmo-unified-data-policy
```

Reply:
@tomkralidis we can not hardcode WMO as contact into the base-api-config, I prefer to leave the contact empty, I have implemented the license as discussed

Reply:
Updated proposal (difference being Apache 2.0 as license):

```yaml
    identification:
        title:
            en: WIS2 in a box API
        description:
            en: |-
                WIS 2.0 in a box (wis2box) provides a platform with the capabilities
                for centres to integrate their data holdings and publish them
                to the WMO Information System with a plug and play capability
                supporting data publishing, discovery and access.
                wis2box-api provides an API providing data and processes for interactive
                visualization, data transformation and publication.
        keywords:
            en:
                - wis2box
                - WIS2 Node
        url: https://docs.wis2box.wis.wmo.int
    provider:
        name: WIS 2.0 in a box
        url: https://docs.wis2box.wis.wmo.int
    contact:
        name: World Meteorological Organization
        address: 7bis, avenue de la Paix
        city: Geneva
        postalcode: 1211
        country: Switzerland
        email: null
        url: https://wmo.int/contact-us
        role: pointOfContact
    license:
        name: Apache License 2.0
        url: https://www.apache.org/licenses/LICENSE-2.0
```